testdata/test_config_two

```diff
@@ -6,4 +6,4 @@ path /static StaticHandler {
 
 path /echo EchoHandler {}
```

Comment:
we need that because that is the other server that we are querying (and asking for echo)